Team18Project/app/src/main/java/com/example/team18project/QRCode.java

```diff
@@ -110,8 +111,27 @@ public String getVisual() { //TODO implement proper representation, maybe change
         return ":)";
     }
 
-    public String getName() { //TODO generate name properly
-        return "QRmon";
+    /**
+     * @return a unique name
```

Comment:
Javadocs should have a line describing what the method does separate from the @return